src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -78,4 +79,55 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workf
           APIResponse.fail(ex))
     }
   }
+
+  //TODO: With PBE, this should be defined in the backend.
```

Comment:
I wanted to move it to the backend trait, but didn't to just have a single file change for this PR. If you think it's okay I don't mind moving it to the backend trait right now.